### ISSUE TITLE
Cross-references with hyperlinks

### ISSUE THREAD

**Issue by jhidding** [entangled/filters#4]
Make references in code clickable, linking to their definition.

**Comment by MyriaCore** [comment]
This would be in `pandoc-anotate-codeblocks`, correct? I'd be interested in this functionality as well.

**Comment by MyriaCore** [comment]
This would be in `pandoc-anotate-codeblocks`, correct? I'd be interested in this functionality as well.

It would be nice if a list of references could also be supplied for a single definition.

**Comment by jhidding** [comment]
Indeed, this does require doing the conversion of code to html in the filter instead of letting pandoc take care of it. I believe `pygments` is the module to do this. I don't know if we can configure it to do a pre-pass highlighting all the Noweb references...
It seems `pygments` has [custom filter support](https://pygments.org/docs/filters/).

**Comment by numero-744** [comment]
We would also need to be able to add references in LaTeX for PDFs, right? Is it additional work or is the pandoc API the same?
